.env.template

```diff
@@ -7,9 +7,22 @@ FAST_LLM_MODEL=gpt-3.5-turbo
 GOOGLE_API_KEY=
 CUSTOM_SEARCH_ENGINE_ID=
 USE_AZURE=False
+OPENAI_API_BASE=your-base-url-for-azure
+OPENAI_API_VERSION=api-version-for-azure
+OPENAI_DEPLOYMENT_ID=deployment-id-for-azure
+WEAVIATE_HOST="127.0.0.1"
+WEAVIATE_PORT=8080
+WEAVIATE_PROTOCOL="http"
+USE_WEAVIATE_EMBEDDED=False
+WEAVIATE_EMBEDDED_PATH="/home/me/.local/share/weaviate"
+WEAVIATE_USERNAME=
+WEAVIATE_PASSWORD=
+IMAGE_PROVIDER=dalle
+HUGGINGFACE_API_TOKEN=
 OPENAI_AZURE_API_BASE=your-base-url-for-azure
 OPENAI_AZURE_API_VERSION=api-version-for-azure
 OPENAI_AZURE_DEPLOYMENT_ID=deployment-id-for-azure
 IMAGE_PROVIDER=dalle
-HUGGINGFACE_API_TOKEN=
```

Comment:
Undo the relocation of the HUGGINGFACE line

Reply:
thanks! there were also some dupes I have now removed.